setup.cfg

```diff
@@ -30,11 +30,12 @@ install_requires =
     ipywidgets>=8.0
     jupyterlab>=4.2.5,<5
     matplotlib>=3.6,<4
+    seaborn>=0.13.2,<0.14
     pandas>=1.5.0,<3
     scikit-learn>=1.2.0,<2
     pyarrow>=9.0.0
     pydantic>=2.6.3,<3
-    ydata-profiling>=4.8.3,<5
+    ydata-profiling>=4.8.3,<4.10
```

Comment:
4.10 breaks some things with `sm.target_feature_summary()`

We get a new error from ydata-profiling about the `Readmitted within 30 Days_Time` column
```text
Types for Readmitted within 30 Days_Time are not compatible: {'DateTime', 'Unsupported'}
```

Reply:
Can you add something to documentation on this need?  
I'm not sure what precedent to set here, but we can't update pypi to have the tighter restriction so it could at least tide us over until releasing the next version

Reply:
added to the README.md so that pip install makes it clear this is needed.